src/main/java/com/littleBeasts/entities/Player.java

```diff
@@ -2,50 +2,60 @@
 
 import calculationEngine.battle.Battle;
 import calculationEngine.entities.*;
+import com.littleBeasts.GameLogic;
 import com.littleBeasts.PlayerState;
+import com.littleBeasts.abilities.Attack;
 import config.HudConstants;
 import de.gurkenlabs.litiengine.Game;
 import de.gurkenlabs.litiengine.IUpdateable;
 import de.gurkenlabs.litiengine.entities.*;
+import de.gurkenlabs.litiengine.graphics.animation.Animation;
 import de.gurkenlabs.litiengine.input.KeyboardEntityController;
+import de.gurkenlabs.litiengine.physics.Force;
+import de.gurkenlabs.litiengine.resources.Resource;
+import de.gurkenlabs.litiengine.resources.Resources;
 
+import java.awt.geom.Point2D;
 import java.util.ArrayList;
 import java.util.List;
 
 @EntityInfo(width = 16, height = 16)
 @MovementInfo(velocity = 70)
 @CollisionInfo(collisionBoxWidth = 14, collisionBoxHeight = 14, collision = true)
 @CombatInfo(hitpoints = 100, team = 1)
-public class Player extends Creature implements IUpdateable {
+public class Player extends Creature implements IUpdateable, IMobileEntity {
     private static Player instance;
     private static PlayerState state = PlayerState.CONTROLLABLE;
     private List<Beast> littleBeastTeam;
     private boolean spawned;
     private Battle battle;
+    private GameLogic gameLogic;
 
 
     private CePlayer cePlayer;
     private CeEntity ceEntity;
-    private Attack[] playerAttacks;
+    private CeAttack[] playerCeAttacks;
     private String playerName = "Horst";
     private int maxHP, currentHP;
     private boolean isFighting;
 
+    private final Attack punch;
```

Comment:
I guess this is also only for dev purposes?

Reply:
No, it's the punch